### DISCUSSION TITLE
How to bring up setup for QEMU with custom ports

### DISCUSSION THREAD

**Discussion by lkusam** [NginxProxyManager/nginx-proxy-manager#1691] [Q&A]
Hello,

I am trying to install the docker on a QEMU. 
I have changed the ports from the host to the following 

`'8080:80'
      - '8081:81'
      - '8443:443'`

This is because the the host doesn't let you use the ports under 1024. 
After the installation docker log says Renew complete and no logs after that. 
I'm trying to use  port 8081 as it is routing to 81. But  setup is not coming up. Is there something I need to change or is there a different way to install for QEMU. 

I know the numbers are stupid. Just put them together to see if it is working.
Thank you for the help

**Comment by chaptergy** [comment]
It if has anything to do with qemu this is probably the wrong place to ask. It could either be that qemu isolates the network and prevents you from accessing it. To see if the issue lies with qemu or npm, go ahead and connect into the docker container so you have a shell inside the container, and run 
```
curl localhost:81
```
If you receive a html string containing `<title>Nginx Proxy Manager</title>`, npm is running correctly and qemu is the issue. You should try and get support somewhere else, specific to virtualization with qemu. If you receive a different response, you can continue here.

Regardless if you found a fix for your issue, you can link it here as the answer.

**Reply by lkusam** [reply]
Thank you @chaptergy . What is final docker log we get after the server started. For me the log is stuck on "Renew Complete".

**Reply by chaptergy** [reply]
*Stuck* is the wrong word here, if you don't do anything on the web interface, then there should not be any more log entries after the "Renew complete". Which is what's happening. But have you tried what I suggested in my comment?
